uyuni-docs-helper

```diff
@@ -66,6 +66,13 @@ print_incorrect_syntax() {
   exit 1
 }
 
+get_dir_abs_path() {
+  (
+    cd "$(dirname ${1})"
```

Comment:
```suggestion
    cd "$(dirname ${1})" || exit
```
See https://www.shellcheck.net/wiki/SC2164

Reply:
This sound be `|| return` not `|| exit`, but since there's a check for the directory before calling this function, I don't really think we need it here.

Reply:
Sure, `|| return` would be more suitable. But feel free to ignore that suggestion :wink:

Reply:
Still, good point @nodeg. I will keep it in mind for the future ;-)